src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/Implementation/FrameworkReferenceAssemblyAttachedCollectionSourceProvider.cs

```diff
@@ -38,7 +38,9 @@ public FrameworkReferenceAssemblyAttachedCollectionSourceProvider(IRelationProvi
                 EnvDTE.Properties? props = projectItem?.Properties;
 
                 if (props?.Item("TargetingPackPath")?.Value is string path &&
-                    props?.Item("OriginalItemSpec")?.Value is string name)
+                    props?.Item("OriginalItemSpec")?.Value is string name &&
+                    !string.IsNullOrWhiteSpace(path) &&
```

Comment:
Nit: This can just be name.Length check. While this whitespace check will probably never be hit, what NTFS considers whitepace and what string.IsNullOrWhitespace considers whitespace are not the same.

Reply:
I used `IsNullOrWhiteSpace` here as there's another test deeper down in the call that does the same. We would use `IsNullOrEmpty` throughout if you think it's likely that a .NET white space string would be a valid path in NTFS.